### PR TITLE
feature(unlock-app, locksmith, unlock-js): Single Stripe account

### PR DESCRIPTION
**Pull request by julien51**
# Description

Letting lock managers re-use a single stripe account for all their locks.

Refs #11555

# Checklist:

- [X] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [X] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: locksmith/src/operations/stripeOperations.ts [hunk 1]
```diff
@@ -135,54 +136,74 @@ export const disconnectStripe = async ({
 }
 
 /**
- * @deprecated
```
Comment (julien51) [inline locksmith/src/operations/stripeOperations.ts:138]: I don't think this is deprecated @searchableguy ? It is the method used by our main flow to connect Stripe accounts...

Comment (searchableguy) [reply locksmith/src/operations/stripeOperations.ts:138]: We have a v2 version. This is deprecated. Check v2 folder.

Comment (searchableguy) [reply locksmith/src/operations/stripeOperations.ts:138]: smh, we have too many connectStripe

https://github.com/unlock-protocol/unlock/blob/173cb49f9bc07bbc15b854639061ba87a94a6d64/locksmith/src/controllers/lockController.ts#L10

File: locksmith/src/controllers/v2/stripeController.ts [hunk 2]
```diff
@@ -36,3 +41,20 @@ export const connectStripe = async (req: Request, res: Response) => {
...
+
+export const getConnectionsForManager = async (req: Request, res: Response) => {
+  const userAddress = req.user!.walletAddress
+  const connections = await stripeOperations.getConnectionsForManager(
+    userAddress
+  )
+  if (!connections) {
+    return []
+  }
+  return res.json(
```
Comment (searchableguy) [inline locksmith/src/controllers/v2/stripeController.ts:53]: want to send something like `result`. We use this everywhere to be consistent so if we have any error but not complete failure, we could attach it as error or additional information later without changing the format.

Comment (julien51) [reply locksmith/src/controllers/v2/stripeController.ts:53]: yes I can! Good point. Thanks!

File: unlock-app/src/components/interface/locks/Settings/forms/CreditCardForm.tsx [hunk 3]
```diff
@@ -124,11 +129,20 @@ export const CreditCardForm = ({
     }
   )
 
+  const {
+    isFetching: isFetchingPreviouslyConnectedLocks,
+    data: previouslyConnectedLocks,
+  } = useQuery(['connectedLocks'], async () => {
```
Comment (searchableguy) [inline unlock-app/src/components/interface/locks/Settings/forms/CreditCardForm.tsx:135]: Pass the account since it is dependent on account for cache.

**Review by searchableguy — APPROVED** [review]

**Pull request merged**